File: app/src/main/java/in/ac/bvmengineering/udaan2k19/DataClass/Manager.java
package in.ac.bvmengineering.udaan2k19.DataClass;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Manager implements Serializable {
    @SerializedName("name")
    private String name;
    @SerializedName("phone")
    private String phone;

    public String getName() {
        return name;
    }

    public String getPhone() {
        return phone;
    }

    @android.support.annotation.NonNull
    @Override
    public String toString() {
        return "Manager{" +
                "name='" + name + '\'' +
                ", phone='" + phone + '\'' +
                '}';
    }
}

File: app/src/main/java/in/ac/bvmengineering/udaan2k19/DataClass/Manager.kt
package `in`.ac.bvmengineering.udaan2k19.DataClass

import com.google.gson.annotations.SerializedName
import java.io.Serializable


class Manager : Serializable {
    @SerializedName("name")
    val name: String? = null
    @SerializedName("phone")
    val phone: String? = null

    override fun toString(): String {
        return "Manager{" +
                "name='" + name + '\''.toString() +
                ", phone='" + phone + '\''.toString() +
                '}'.toString()
    }
}